test/unit_tests/graphql/test_subscription.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 SURF.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from http import HTTPStatus
+from uuid import uuid4
+
+
+def build_simple_query(subscription_id):
+    q = f"""
```

Comment:
I think we can make these queries a lot more readable by using graphql variables (like we do in Pythia)

Reply:
create a issue to update this :3

Reply:
Why? This is a small change